.buildkite/longruns_gpu/pipeline.yml

```diff
@@ -95,6 +83,36 @@ steps:
         env:
           CLIMACOMMS_DEVICE: "CUDA"
 
+  - group: "Perturbed Low-res Global Land Models"
+    if: build.env("PERTURBED_RUN") != null
```

Comment:
how does this var get set?

Reply:
you set it in buildkite - click on new build in the top right corner, and then add your branch instead of main (or use main), and then add an environment variable
PERTURBED_RUN=""

(it is also how we trigger longer runs if you want to do a one off run of those. Otherwise we set the environment variable in our scheduled long run here: https://buildkite.com/clima/climaland-long-runs/settings/schedules/c7373c40-27cc-4051-b13a-58769f62f5e7)